README.md

```diff
@@ -1,132 +1,106 @@
 
-### Prerequisites:
-- a  Linkurious Enterprise 2.9.x server
+![](readme_assets/plugin.png)
 
-## Topics:
-1. [Introduction](#1-introduction)
+### Prerequisites
+- Linkurious Enterprise 2.9.x or above
+- CSV files containing nodes and edges structured as follows:
 
-2. [Installation](#2-installation)
-    1. [Setting the Plugin](#21-setting-the-plugin)
-    2. [Setting the Custom Action](#22-setting-the-custom-action)
+#### for nodes
 
-3. [How to import data](#3-how-to-import-data)
+![](src/public/assets/img/node.png)
 
-4. [Sample files](#4-sample-files)
+#### for edges
 
-## 1. Introduction
-With this plugin, we want to provide an easy tool for importing data into your datasource, even if you're not a cypher query language expert: no knowledge is required.
+![](src/public/assets/img/edge.png)
 
-The information of nodes and edges need to be stored in a *comma separated file* (also known as *csv* file); every single line of the file should follow this syntax:
-
-```
-property_1,property_2,...,property_n
-```
-
-where `property_x` is:
-- a `number`: an integer or a float (`.` must be used as decimal separator);
-- a `string`: they can be defined with or without `"` (or `'`) delimiters;
-- a `date` (or `datetime`);
-- a `boolean`;
-
-NOTE: regardless of the type, at the moment it's not possible to include a `,` in the value of a property.
-
-It's possible to include an *header* containing the names of the property. The header must be the first line of the file and it follow the same syntax as the property values.
-
-Example: *sample.csv*
-
-```
-first_name,last_name,age,country
-Roosvelt,Anderson,42,"US"
-Julianne,Redmond,35,"GB"
-Ginevra,Spacey,87,"AU"
-```
-
-## 2. Installation
-
-### 2.1 Setting the Plugin
-NOTE: for this part you need admin rights.
-
-1. Download the `importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
-
-2. Copy the `.lke` archive in the folder `<lke-server>/data/plugins`
+Two sample CSV files, for nodes and edges, are available [here](https://github.com/Linkurious/lke-plugin-csv-importer/tree/master/sample%20csv):
+- `person.csv` contains information about person entities (a *node*);
+- `ping.csv` contains information about relationships between two person nodes (a *relationship*).
 
-3. On Linkurious Enterprise dashboard, go to `Admin` -> `Global configuration`
+## Set up the Plugin
 
-4. Scroll to the `Plugin settings` field
+> Make sure to have write access to the folder of Linkurious Enterprise as well as a Linkurious admin account.
 
-5. If there are no other plugins configured here, replace the whole content of the field with the following code and skip to step 7. \
-Otherwise, go to step 6
+1. Download the `csv-importer.lke` plugin available [here](https://github.com/Linkurious/lke-plugin-csv-importer/releases/tag/v1.0.0).
+2. Copy `csv-importer.lke` file to the following path `<linkurious>/data/plugins`
+3. Restart Linkurious Enteprise
```

Comment:
in the step 6 the user will update the config, so maybe there is no need to restart linkurious here ?

or leave this step and remove the last one as the default config will be added automatically